1.2.x fails to build with error: expected a type

# Bug

After updating to 1.2.0 from 1.1.1, the module fails to build in Xcode with the following error:

```
RNCPushNotificationIOS.h:22:41: error: expected a type  + (void)didReceiveNotificationResponse:(UNNotificationResponse *)response API_AVAILABLE(ios(10.0));
```

## Environment info

`react-native info` output:

```bash
info
  React Native Environment Info:
    System:
      OS: macOS 10.15.3
      CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
      Memory: 1.29 GB / 16.00 GB
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 12.4.0 - ~/.nvm/versions/node/v12.4.0/bin/node
      Yarn: 1.5.1 - /usr/local/bin/yarn
      npm: 6.9.0 - ~/.nvm/versions/node/v12.4.0/bin/npm
    SDKs:
      iOS SDK:
        Platforms: iOS 12.4, macOS 10.14, tvOS 12.4, watchOS 5.3
    IDEs:
      Android Studio: 3.6 AI-192.7142.36.36.6241897
      Xcode: 10.3/10G8 - /usr/bin/xcodebuild
    npmPackages:
      react: 16.13.1 => 16.13.1
      react-native: 0.59.10 => 0.59.10
```

Library version: 1.2.0

--- a/ios/RNCPushNotificationIOS.h
+++ b/ios/RNCPushNotificationIOS.h
@@ -6,6 +6,7 @@
  */
 
 #import <React/RCTEventEmitter.h>
+#import <UserNotifications/UserNotifications.h>
 
 extern NSString *const RNCRemoteNotificationReceived;
 

